templates/s3-bucket.yaml

```diff
@@ -79,6 +88,21 @@ Resources:
               Resource: !Sub ${Bucket.Arn}/*
             - !Ref AWS::NoValue
 
+  SynapseOwnerFile:
+    Type: AWS::S3::Object
+    Condition: ConnectToSynapse
```

Comment:
You removed the condition up top.  You will have to change this.

Reply:
Hmm, do these lines not cover it: [L32-L33](https://github.com/Sage-Bionetworks/recover/blob/3c134df081085a0e58a0e3a292ac8a97462ae163/templates/s3-bucket.yaml#L32-L33)?

Reply:
Ah my mistake! Thanks!